.github/copilot-instructions.md

```diff
@@ -0,0 +1,145 @@
+
+        PRO: https://github.com/obophenotype/pro_obo_slim
+        CHEBI: https://github.com/obophenotype/chebi_obo_slim
+
+## OBO Format Guidelines
+- Term ID format: OBA:NNNNNNN (7-digit number)
+- Handling New Term Requests (NTRs):
+  - New terms start OBA:210xxxx
+  - Do `grep id: OBA:210 src/ontology/oba-edit.obo` to check for clashes
+- Each term requires: id, name, definition with references
+- Never guess OBA IDs, or ontology term IDs, use search tools above to determine actual term
+- Never guess PMIDs for references, do a web search if needed
+- Use standard relationship types: is_a, part_of, has_part, etc.
+- Follow existing term patterns for consistency
+
+## Publications
+- Run the command `aurelian fulltext <PMID:nnn>` to fetch full text for a publication. A doi or URL can also be used
+- You should cite publications appropriately, e.g. `def: "...." [PMID:nnnn, doi:mmmm]
+
+## GitHub Contribution Process
+- Most requests from users should follow one of two patterns:
+    - You are not confident how to proceed, in which case end with asking a clarifying question (via `gh`)
+    - You are confident how to proceed, you make changes, commit on a branch, and open a PR for the user to review
+- Check existing terms before adding new ones
+- For new terms: provide name, definition, place in hierarchy, and references
+- Include PMIDs for all assertions
+- Follow naming conventions from parent terms
+- Always commit in a branch, e.g. issue-NNN
+- If there is an existing PR which you started then checkout that branch and continue, rather than starting a new PR (unless you explicitly want to abandon the original PR, e.g. it was on completely the wrong tracks)
+- Always make clear detailed commit messages, saying what you did and why
+- Always sign your commits `@dragon-ai-agent`
+- Create PRs using `gh pr create ...`
+- File PRs with clear descriptions, and sign your PR
+
+## Handling GitHub issues and requests
+- Use `gh` to read and write issues/PRs
+- Sign all commits and PRs as `@dragon-ai-agent`
+
+## TROUBLESHOOTING
+- If your obo file has syntax errors, you can use `robot convert -vvv` to see full trace
+
+## Obsoleting terms originating from DOS-DP templates
+
+- Delete the row with the targeted term from the appropriate data table (tsv file) in folder: src/patterns/data/default/
+- Fill in the IRI of the term to be obsoleted in src/templates/obsoletes.tsv along with other relevant information. It is useful add a comment with an explanation for the obsoletion. Start the comment with 'Reason for obsoletion: '.
+- Re-create the DOS-DP axioms by running
+
+    cd PATH-TO/bio-attribute-ontology/src/ontology
+    sh run.sh make ../patterns/definitions.owl
+    sh run.sh make components/obsoletes.owl
```

Comment:
```suggestion
    make components/obsoletes.owl
```

Remove run.sh and see if ROBOT alone works for  this (the robot instealled in the setup steps). Ask me if any problems.